source/frontend/connect4.js

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable no-loop-func */
+/* eslint-disable no-use-before-define */
+
+let gameEnded = false;
+
+player1 = prompt('player1: ', 'Red');
+player2 = prompt('player2: ', 'Yellow');
+
+/* Display the same board - when browser refreshes */
+$.ajax({
+  type: 'GET',
+  url: '/get-board',
+  contentType: 'application/json',
+  success: (result) => {
+    drawBoard(result.board);
+
+    $('h3').text(`${player1}: it is your turn, please pick a column to drop your chip.`).show();
+
+    const playerObj1 = result.scores.filter((item) => item.player === 1)[0];
```

Comment:
You could also use `.find` for this
```suggestion
    const playerObj1 = result.scores.find((item) => item.player === 1);
```